dist/info/moonlight_libretro.info

```diff
@@ -0,0 +1,23 @@
+## All data is optional, but helps improve user experience.
+
+# Software Information - Information about the core software itself
+display_name = "Moonlight"
+categories = "Streaming client"
+authors = "Rock88"
+corename = "Moonlight"
+license = "GPLv3"
+display_version = "1.0.3"
+
+# Hardware Information - Information about the hardware the core supports (when applicable)
+manufacturer = "nVidia"
```

Comment:
It's branded as NVIDIA in most places on their website :shrug: 

``` suggestion
manufacturer = "NVIDIA"
```

Reply:
oh yea missed that from your first comment. updated again.